From 76bd656f77bc00a5cc2d668742e8bad3984c0ab8 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:49 -0400
Subject: [redhat] kabi: Adding symbol call_usermodehelper_exec (kernel/kmod.c)

Message-id: <44b186ff9d4db1c8ca1e5730d64b831b23a298ae.1410510773.git.poros@redhat.com>
Patchwork-id: 95916
O-Subject: [PATCH RHEL7.1 049/233] kabi: Adding symbol call_usermodehelper_exec (kernel/kmod.c)
Bugzilla: 1071097
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Start a usermode application.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_exec
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_exec
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_exec

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_exec b/redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_exec
new file mode 100644
index 0000000..4c5ffeb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_exec
@@ -0,0 +1,2 @@
+#1-
+0x93a8d611 call_usermodehelper_exec vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_exec b/redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_exec
new file mode 100644
index 0000000..b7d24db
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_exec
@@ -0,0 +1,2 @@
+#1-
+0x13996a7d call_usermodehelper_exec vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_exec b/redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_exec
new file mode 100644
index 0000000..26fb828
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_exec
@@ -0,0 +1,2 @@
+#1-
+0xe85b5e75 call_usermodehelper_exec vmlinux EXPORT_SYMBOL
-- 
1.7.1